### PR TITLE
UIDATIMP-217: add rowUpdater to fix checkboxes

### PR DESCRIPTION
**Pull request by OleksandrYehorovEPAM**

### CODE REVIEW

File: src/components/ListView/ListView.js [hunk 1]
```diff
@@ -165,6 +165,12 @@ export class ListView extends Component {
     />
   );
 
+  rowUpdater = ({ id }) => {
+    const { checkboxList: { selectedRecords } } = this.props;
+
+    return selectedRecords && selectedRecords.has(id);
```
Comment (VictorSoroka1) [inline src/components/ListView/ListView.js]: I see that for similar cases below you have used just 
`return selectedRecords.has(id);`

Is there additional check is needed?

Comment (OleksandrYehorovEPAM) [reply src/components/ListView/ListView.js]: The check is needed because `checkboxList` is not a required property. However, I am going to refactor it.

Comment (VictorSoroka1) [reply src/components/ListView/ListView.js]: Sounds good.

**Comment by VictorSoroka1** [comment]
@OleksandrYehorovEPAM, please also add a note to the changelog file.

**Pull request merged**